apps/website/src/components/chart/components/chart-tooltip.tsx

```diff
@@ -0,0 +1,122 @@
+import { animated } from '@react-spring/web'
+import { Text } from '@status-im/components'
+import { DoneIcon, OpenIcon } from '@status-im/icons'
+import { Stack } from '@tamagui/core'
+import { defaultStyles, TooltipWithBounds } from '@visx/tooltip'
+
+import type { SpringValue } from '@react-spring/web'
+
+const AnimatedTooltip = animated(TooltipWithBounds)
+
+type Props = {
+  tooltipData: {
+    totalIssues: number
+    closedIssues: number
+    date: string
+    formattedDate: string
+    openIssuesPercentage: number
+    completedIssuesPercentage: number
+    openIssues: number
+  }
+  opacityAnimation: {
+    opacity: SpringValue<number>
+  }
+  tooltipAnimation: {
+    x: SpringValue<number>
+    y: SpringValue<number>
+  }
+}
+
+// defining tooltip styles
+const tooltipStyles = {
```

Comment:
Please, try to use types for any arbitrary defined objects, so TypeScript helps us in the future catch any mistakes.
```suggestion
const tooltipStyles: React.CSSProperties = {
```